share/sv.po

```diff
@@ -12,7 +12,7 @@ msgstr ""
 msgid "Print version information and exit."
 msgstr "Skriv ut versionsinformation och avsluta."
 
-msgid "The minimum severity level to display"
+msgid "The minimum severity level to display."
```

Comment:
This is not correct msgid from the code.

Reply:
Yes, my mistake. And thanks for the tip with the make command ;)